config.js

```diff
@@ -650,7 +650,9 @@ assert(config.NAMESPACE_CACHING.DEFAULT_BLOCK_SIZE > config.INLINE_MAX_SIZE);
 //////////////////
 
 config.NSFS_BUF_SIZE = 8 * 1024 * 1024;
-config.NSFS_BUF_POOL_MEM_LIMIT = config.BUFFERS_MEM_LIMIT;
+config.NSFS_BUF_POOL_MEM_LIMIT = config.ENDPOINT_FORKS > 0 ?
```

Comment:
hmmm why this is the final change? asking because the JIRA description specifies having 4 gb memory limit, 3 CPU limit per fork

Reply:
added the 4GB limit